pages/home.page.ts

```diff
@@ -1,6 +1,29 @@
-import { Locator } from "@playwright/test";
-import { BasePage } from "./base.page";
+import { Locator } from '@playwright/test';
+import { BasePage } from './base.page';
 
-export class HomePage extends BasePage{
-    readonly productCard: Locator = this.page.getByTestId('product-name');
-}
\ No newline at end of file
+export class HomePage extends BasePage {
+  readonly productTitle: Locator = this.page.getByTestId('product-name');
+  readonly productPrice: Locator = this.page.getByTestId('product-price');
+  readonly sortSelect: Locator = this.page.getByTestId('sort');
+
+
+  async selectSortOption(value: string) {
+        await this.sortSelect.click();
+        await this.sortSelect.selectOption(`${value}`);
+        await this.page.waitForLoadState('domcontentloaded');
+        //before adding additional time, waiting for load state seemed to be not enough
+        await this.page.waitForTimeout(1000);
+    };
+
+    async selectCategoryCheckbox(name: string) {
```

Comment:
```suggestion
    async selectCategoryCheckbox(name: string): Promise<void> {
```
and in other methods